server/routes/user.js

```diff
@@ -28,4 +28,19 @@ router.post(
   }
 );
 
+router.put("/:id/experience", async (req, res) => {
+  // Get language data from body of request, userID from params
+  const languages = req.body;
+  const user = await User.findById(req.params.id);
+  try {
+    // loop through body of request, setting user experience for each key
+    for (let language in languages) {
+      await user.experience.set(language, Number(languages[language]));
```

Comment:
maybe we should validate languages against a predefined list

Also, is there a way to set it all at once as opposed to doing n calls to the database to set this information on the user? It's not that efficient